app/BUILD.bazel

```diff
@@ -10,23 +10,24 @@ order to build with Bazel.
 - All binding adapters must be written in Java.
 """
 
-load("@rules_jvm_external//:defs.bzl", "artifact")
 load("@dagger//:workspace_defs.bzl", "dagger_rules")
+load("@io_bazel_rules_kotlin//kotlin:kotlin.bzl", "kt_android_library")
+load("@rules_android//android:rules.bzl", "android_library")
+load("@rules_jvm_external//:defs.bzl", "artifact")
 load("@tools_android//tools/crashlytics:defs.bzl", "crashlytics_android_library")
 load("@tools_android//tools/googleservices:defs.bzl", "google_services_xml")
-load("@io_bazel_rules_kotlin//kotlin:kotlin.bzl", "kt_android_library")
 load("//app:app_test.bzl", "app_test")
 load("//app:test_with_resources.bzl", "test_with_resources")
 
-"""
-Source files for the annotations library. The files inside the annotations library are dependencies
-for all other libraries in app module, except the listeners library.
-Place your file here if:
- - A view, view model, or layout file depends on it
- - It is an annotation or helper file of some kind
- - It is not considered a listener
- - It does not depend on any other file not included in this list
-"""
+package(default_visibility = ["//visibility:public"])
```

Comment:
Continuing the Ben's comment here

Reply:
We should not default visibility to public by default. Library visibility should opt into being available, not opt out of it.